exonum/src/helpers/fabric/details.rs

```diff
@@ -554,8 +574,20 @@ impl Command for Finalize {
                 ConfigFile::load(path).expect("Failed to load validator public config.")
             })
             .collect();
+
         let (common, list, our) = Self::reduce_configs(public_configs, &secret_config);
 
+        let validators_count = common
+            .general_config
+            .get("validators_count")
+            .expect("validators_count is not found in common config.")
```

Comment:
Why not make it optional with default value ? Just to avoid breaking changes.

Reply:
@DENMROOT I'm not sure that we want to "avoid breaking changes", at least for now. :upside_down_face: This will require some compromises and introduce additional complexity.

Additionally, as far as I know, we need this parameter to be explicit in order to prevent mistakes. With default value we are going to lose that advantage.